### PR TITLE
Add detail label for autolock setting

### LINKED ISSUE #188: Display saved user preference for "Auto Lock" on settings screen
A minor follow-up to #168 and #80

Output the stored user preference in the settings view cell itself so users don't have to tap in to the setting to see what it's set to

https://app.zeplin.io/project/5a4e9ff8032d0923200a2635/screen/5a4ea092a1571f12f4d05cd8

settings_-_touch_id_enabled_-_lockbox_-_ios_-_zeplin

### PR DESCRIPTION
**Pull request by joeyg**
Resolves #188.

simulator screen shot - iphone 8 - 2018-04-06 at 14 55 37

### CODE REVIEW

File: lockbox-ios/View/SettingsView.swift [hunk 1]
```diff
@@ -76,11 +76,13 @@ extension SettingsView {
     private func setupDataSource() {
         self.dataSource = RxTableViewSectionedReloadDataSource(
                 configureCell: { _, _, _, cellConfiguration in
-                    let cell = UITableViewCell()
-                    cell.textLabel?.text = cellConfiguration.text
+
+                let cell = UITableViewCell(style: UITableViewCellStyle.value1, reuseIdentifier: "settings-cell")
```
[the remark this answers is not in the document]
Comment (joeyg) [reply lockbox-ios/View/SettingsView.swift]: These two lines were indented one additional tab than the lines below it. Should the rest of the function be changed to match the further indented style?

**Comment by devinreams** [comment]
@joeyg I think BuddyBuild got stuck somehow and wasn't running tests against your PR here.

I just manually ran the suite and see the `_displayTemporaryAlert__displays_a_UIView_temporarily` spec is failing: https://dashboard.buddybuild.com/apps/5a0ddb736e19370001034f85/build/5acbdd161bb7a60001ff64ff/test/2a82dc1b-833d-465a-a9a8-9d3c725a1ced/79885946-34f2-4fe5-bbd5-5c13d27bd3f3

**Comment by joeyg** [comment]
@devinreams does this happen every time? I ran the tests locally and they all passed in this branch.

screen shot 2018-04-10 at 10 28 27 am

**Comment by devinreams** [comment]
>does this happen every time? I ran the tests locally and they all passed in this branch.

Odd, I re-built without cache and seems to have resolved itself. ✅

Sorry for the interrupt and thanks for checking on your side. I should've re-attempted that myself first.

**Comment by joeyg** [comment]
@devinreams No problem. Did you happen to figure out why buddy build got stuck initially? Does it not build PRs from forks?

**Comment by devinreams** [comment]
>Did you happen to figure out why buddy build got stuck initially? Does it not build PRs from forks?

Not by default, but I did find the setting! 😁

**Comment by devinreams** [comment]
Thanks again for your contributions here, @joeyg. The settings are really coming together nicely 😍

**Pull request merged**